ci-operator/config/openshift/cluster-cloud-controller-manager-operator/openshift-cluster-cloud-controller-manager-operator-master.yaml

```diff
@@ -3,12 +3,23 @@ base_images:
     name: "4.7"
     namespace: ocp
     tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.7
 binary_build_commands: go build cmd/cluster-cloud-controller-manager-operator/main.go
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
     tag: golang-1.15
+images:
+- from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7
+  to: cluster-cloud-contrller-manager-operator
```

Comment:
This name is not right. It also should be simply `cluster-cloud-controller-manager`, as is in https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/master/manifests/image-references#L8

Reply:
I think it should be called `cluster-cloud-contrller-manager-operator` to comply with other components